etc/c/sqlite3.d

```diff
@@ -1,35 +1,35 @@
 module etc.c.sqlite3;
 /*
-** 2001-09-15
-**
-** The author disclaims copyright to this source code.  In place of
-** a legal notice, here is a blessing:
-**
-**    May you do good and not evil.
-**    May you find forgiveness for yourself and forgive others.
-**    May you share freely, never taking more than you give.
-**
+* 2001-09-15
+*
+* The author disclaims copyright to this source code.  In place of
+* a legal notice, here is a blessing:
+*
+*    May you do good and not evil.
+*    May you find forgiveness for yourself and forgive others.
+*    May you share freely, never taking more than you give.
+*
 *************************************************************************
-** This header file defines the interface that the SQLite library
-** presents to client programs.  If a C-function, structure, datatype,
-** or constant definition does not appear in this file, then it is
-** not a published API of SQLite, is subject to change without
-** notice, and should not be referenced by programs that use SQLite.
-**
-** Some of the definitions that are in this file are marked as
-** "experimental".  Experimental interfaces are normally new
-** features recently added to SQLite.  We do not anticipate changes
-** to experimental interfaces but reserve the right to make minor changes
-** if experience from use "in the wild" suggest such changes are prudent.
-**
-** The official C-language API documentation for SQLite is derived
-** from comments in this file.  This file is the authoritative source
-** on how SQLite interfaces are suppose to operate.
-**
-** The name of this file under configuration management is "sqlite.h.in".
-** The makefile makes some minor changes to this file (such as inserting
-** the version number) and changes its name to "sqlite3.h" as
-** part of the build process.
+* This header file defines the interface that the SQLite library
+* presents to client programs.  If a C-function, structure, datatype,
+* or constant definition does not appear in this file, then it is
+* not a published API of SQLite, is subject to change without
+* notice, and should not be referenced by programs that use SQLite.
+*
+* Some of the definitions that are in this file are marked as
+* "experimental".  Experimental interfaces are normally new
+* features recently added to SQLite.  We do not anticipate changes
+* to experimental interfaces but reserve the right to make minor changes
+* if experience from use "in the wild" suggest such changes are prudent.
+*
+* The official C-language API documentation for SQLite is derived
+* from comments in this file.  This file is the authoritative source
+* on how SQLite interfaces are suppose to operate.
+*
+* The name of this file under configuration management is "sqlite.h.in".
+* The makefile makes some minor changes to this file (such as inserting
+* the version number) and changes its name to "sqlite3.h" as
+* part of the build process.
```

Comment:
This is a regular comment, not a doc comment. Why are these changes necessary?

Reply:
Unnecessary—fixed.